### DISCUSSION TITLE
New distributed install - no logs forwarded to manager

### DISCUSSION THREAD

**Discussion by autosoap** [Security-Onion-Solutions/securityonion#2486] [Unsupported Versions]
I set up a new distributed install with a single sensor. Both sensor and manager appear to be running nominally according to so-status and salt-call state.highstate. Sensor is registered with manager and traffic is hitting the bond interface of the sensor but the manager is not receiving any logs. Any suggestions on troubleshooting?

**Comment by dougburks** [comment]
Do you have Zeek enabled on the sensor?

If so, is Zeek writing logs to `/nsm/zeek/logs/current/` on the sensor?

**Comment by autosoap** [comment]
Yes, logs are being written on the sensor.

**Comment by defensivedepth** [comment]
Just to double-check - Are you running a search node?

**Comment by autosoap** [comment]
No, it’s a forward.

**Reply by defensivedepth** [reply]
Right, so do you have at least of of each? A Manager Node, Search Node, Forward Node?

**Comment by autosoap** [comment]
I did only have a Manager node and Forward node. I re-installed the Forward
node as a Heavy node and I'm still having the same issue. Getting logs in
/nsm/zeek/logs/current/ on the Heavy node but nothing on the manager.

**Comment by dougburks** [comment]
Is there a network firewall between the heavy node and the manager? If so, what ports have you allowed through it?

**Comment by autosoap** [comment]
No, there's no firewall between them.

**Comment by dougburks** [comment]
Have you checked the logs in `/opt/so/log/` on both boxes for additional clues?

What's the output of the following on the manager?
```
curl localhost:9200/_cluster/settings?pretty
```
